vision_stack/buoy_detect/src/integration.cpp

```diff
@@ -116,8 +125,14 @@ void imageCallback(const sensor_msgs::ImageConstPtr& msg)
     {
         growhandle.start_grow(eroded, final_image, edgeMap, seed_points[i].x, seed_points[i].y, seed_points[i].color);
     }
+    
+    
     cv_ptr->image = final_image;
     final_pub->publish(cv_ptr->toImageMsg());
+    auto t4 = clk::now();
+    cout << "time re receive and queue : " << duration_cast<milliseconds>(t2 - t1).count() <<endl;
```

Comment:
@pranaypratyush Just follow the normal severity rules. Anything that doesn't hamper information during an actual run will go into `DEBUG`. Information required all the time will go into `INFO`, and errors will go into `ERROR`. ROS handles coloring of messages and tagging with timestamps too, so, makes debugging a lot easier when testing in the pool.